### DISCUSSION TITLE
Ultimaker 2 with SuperSlicer

### DISCUSSION THREAD

**Discussion by Seele86** [supermerill/SuperSlicer#2298] [Q&A]
Hi, 
I am new to the SuperSlicer but I already love it. Even if I couldn't print a single part yet. 
At the moment I am struggeling with the settings for Ultimaker. 
My starting gcode is working, but then the extrusion is much too little. I think the problem are the basic settings in the program. 
Has anyone already tried the SuperSlicer with an Ultimaker 2?

I attached a file which doesn't work properly:
[Silicone_Cap.zip](https://github.com/supermerill/SuperSlicer/files/7981192/Silicone_Cap.zip)

Thanks in advance
Seele86

**Comment by supermerill** [comment]
Did you set the filament to 2.85 mm ? I think all the defaults are at 1.75mm.
mmm, the problem is that you have too little and not too many...

Other than that... be sure to use SI units.

**Reply by Seele86** [reply]
Thanks for your fast reply. 
Yes, I have set the filament diameter to 2,85mm. 
Can I switch the units somewhere in the Superslicer? 
The program itself is set to [mm] but maybe there is a "hidden" menu.

**Reply by supermerill** [reply]
You can convert model to mm. (contextual menu -> convert from imperial units).

I won't mess with units. The whole world engineers & scientists use the SI unit anyway. There is only 4% (minus the ones that go to university) of the world that still use imperial units. If you're one of thoses 4%, as you're in 3d printing, you should be educated enough to use SI unit.

**Reply by Seele86** [reply]
We always use the SI units here, but I thought maybe the Slicer has something internally set to imperial units.
Unfortuantely this wasn't the problem.

**Reply by supermerill** [reply]
try to increase the extrusion multiplier. When you will get the right flow, maybe the multiplier number give you a hint.

**Comment by Seele86** [comment]
This is what I get with a Extrusion Multiplier of 220% and then I raised up the Material Flow while printing to 500%. 
It seems that the extruder is not working while printing. When I grope the filament at the extruder I cannot feel any forwarding.
During the gcode script before the print, the printer is extruding the set 20mm of filament without any problem.

![PXL_20220202_103943449](https://user-images.githubusercontent.com/98843207/152139169-0194dbf5-ea87-4a51-95a9-8c4cb10477ed.jpg)

**Reply by supermerill** [reply]
in the printer, there is a relative / absolute mode checkbox.
Maybe your printer only support one and you have the other selected.

**Comment by Seele86** [comment]
Problem found. 
I looked at my gcode and there are all the extrusions missing. 
[image](https://user-images.githubusercontent.com/98843207/152150887-1e40e4e6-8c3a-4a16-9daa-f312f0834d8a.png)

I was the wrong G-Code flavor 
![image](https://user-images.githubusercontent.com/98843207/152151516-d0636231-ab2e-4155-bd81-983f5c5af8b8.png)

Which one is recommende for the Ultimaker? Does Marlin cause any problems?

**Reply by supermerill** [reply]
sprinter is the most basic one. Marlin should be good if sprinter isn't.

**Reply by Seele86** [reply]
@supermerill thanks a lot. It's working now. I need to get used to the settings of SuperSlicer because I come from Simplify but I am really satisfied with my first complete print.
